test/Operators/finitedifference/column.jl

```diff
@@ -775,6 +775,196 @@ end
+        adv_wc = @. divf2c.(first_order_fluxsinᶠ) + corrected_antidiff_flux
+
+        Δh[k] = cs.face_local_geometry.J[1]
+
+        # Error
+        err_adv_wc[k] = norm(adv_wc .- cos.(centers))
+    end
+
+    # Check convergence rate
+    conv_adv_wc = convergence_rate(err_adv_wc, Δh)
+
+    # Upwind3rdOrderBiasedProductC2F conv, with f(z) = sin(z)
+    @test err_adv_wc[3] ≤ err_adv_wc[2] ≤ err_adv_wc[1] ≤ 5e-4
+    @test conv_adv_wc[1] ≈ 3 atol = 0.1
+    @test conv_adv_wc[2] ≈ 3 atol = 0.1
+    @test conv_adv_wc[3] ≈ 3 atol = 0.1
+    @test conv_adv_wc[1] ≤ conv_adv_wc[2] ≤ conv_adv_wc[2]
+end
+
+@testset "Simple FCT: lin combination of UpwindBiasedProductC2F + Upwind3rdOrderBiasedProductC2F on (uniform and stretched) non-periodic mesh, with FirstOrderOneSided BCs" begin
+    FT = Float64
+    n_elems_seq = 2 .^ (4, 6, 8, 10)
+    stretch_fns = (Meshes.Uniform(), Meshes.ExponentialStretching(1.0))
+
+    for (i, stretch_fn) in enumerate(stretch_fns)
+        err_adv_wc = zeros(FT, length(n_elems_seq))
+        Δh = zeros(FT, length(n_elems_seq))
+        for (k, n) in enumerate(n_elems_seq)
+            domain = Domains.IntervalDomain(
+                Geometry.ZPoint{FT}(-pi),
+                Geometry.ZPoint{FT}(pi);
+                boundary_tags = (:bottom, :top),
+            )
+            mesh = Meshes.IntervalMesh(domain, stretch_fn; nelems = n)
+
+            cs = Spaces.CenterFiniteDifferenceSpace(mesh)
+            fs = Spaces.FaceFiniteDifferenceSpace(cs)
+
+            centers = getproperty(Fields.coordinate_field(cs), :z)
+            C = 1.0 # flux-correction coefficient (falling back to third-order upwinding)
+
+            # UpwindBiasedProductC2F & Upwind3rdOrderBiasedProductC2F Center -> Face operator
+            # Unitary, constant advective velocity
+            w = Geometry.WVector.(ones(fs))
+            # c = sin(z), scalar field defined at the centers
+            Δz = FT(2pi / n)
+            c = (cos.(centers .- Δz / 2) .- cos.(centers .+ Δz / 2)) ./ Δz
+            s = sin.(centers)
+
+            first_order_fluxᶠ = Operators.UpwindBiasedProductC2F(
+                bottom = Operators.Extrapolate(),
+                top = Operators.Extrapolate(),
+            )
+            third_order_fluxᶠ = Operators.Upwind3rdOrderBiasedProductC2F(
+                bottom = Operators.FirstOrderOneSided(),
+                top = Operators.FirstOrderOneSided(),
+            )
+
+            divf2c = Operators.DivergenceF2C(
+                bottom = Operators.SetValue(
+                    Geometry.Contravariant3Vector(FT(0.0)),
+                ),
+                top = Operators.SetValue(
+                    Geometry.Contravariant3Vector(FT(0.0)),
+                ),
+            )
+
+            corrected_antidiff_flux = @. divf2c(
+                C * (third_order_fluxᶠ(w, c) - first_order_fluxᶠ(w, c)),
+            )
+            adv_wc =
+                @. @. divf2c.(first_order_fluxᶠ(w, c)) + corrected_antidiff_flux
```

Comment:
```suggestion
            adv_wc =
                @. divf2c(first_order_fluxᶠ(w, c)) + corrected_antidiff_flux
```

Reply:
Good eye! Sorry that slipped by accident